pkg/gatherers/workloads/workloads_info.go

```diff
@@ -38,13 +38,31 @@ const (
 	podsLimit = 8000
```

Comment:
I just want to add a note here to say that, this gatherer has some constants and global variables that I would LOVE to remove, but it seems a little bit complicated to include on this PR. If I decide to change it, I'll submit it as a separate PR.